package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "chrome-dgram",
   "description": "Use the Node `dgram` API in Chrome Apps",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
Good point, thanks. 

Normally I don't include version changes in PRs. This one slipped through the cracks.